documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/blang/semver"
```

Comment:
I originally intended to do this, main reason I didn't was that, AIUI, people are using the storage adapter from master with 1.6 releases of Prometheus.  Making this change will break everyone who tries to do this.

So perhaps we should make a "release" of the storage adapter?  I'm just trying to avoid lots of issues & emails on the users list.